src/config/ConfigHelper.ts

```diff
@@ -179,6 +202,7 @@ export class ConfigHelper {
         FixedPrice,
         Dispenser,
         ERC721Factory,
+        ERC20Template, // added to Config
```

Comment:
why do we need this? We can fetch the list from ERC721Factory

Reply:
not needed, probably was thinking about something else at the time